hybrid/SampleApps/NoteSync/NoteSync/SFContentSoqlSyncDownTarget.m

```diff
@@ -178,7 +178,6 @@ - (id)initWithQuery:(NSString*)query
     self = [[self class] requestWithMethod:SFRestMethodPOST path:SOAP_PATH queryParams:nil];
     if (self) {
         self.endpoint = SOAP_ENDPOINT;
-        self.parseResponse = NO;
```

Comment:
I removed the `parseResponse` property since it's not required anymore. The new network stack will automatically pass back `NSData` if the response can't be parsed.